### PR TITLE
feat(youtube): `disable-zoom-haptics` patch

### PR DESCRIPTION
**Pull request by aliernfrog**
Disables vibration when zooming videos in YouTube, tested on 17.43.36

https://github.com/revanced/revanced-integrations/pull/217

### CODE REVIEW

File: src/main/kotlin/app/revanced/patches/youtube/misc/zoomhaptics/annotations/ZoomHapticsCompatibility.kt [hunk 1]
```diff
@@ -0,0 +1,13 @@
+package app.revanced.patches.youtube.misc.zoomhaptics.annotations
+
+import app.revanced.patcher.annotation.Compatibility
+import app.revanced.patcher.annotation.Package
+
+@Compatibility(
+    [Package(
+        "com.google.android.youtube", arrayOf("17.43.36")
```
Comment (oSumAtrIX) [inline src/main/kotlin/app/revanced/patches/youtube/misc/zoomhaptics/annotations/ZoomHapticsCompatibility.kt:8]: I believe it is fine to remove the versions array and make it compatible with all future versions. Other than that this should support the targeted, latest version as well.

**Review by oSumAtrIX — CHANGES_REQUESTED** [review]
Looks good.

File: src/main/kotlin/app/revanced/patches/youtube/misc/zoomhaptics/annotations/ZoomHapticsCompatibility.kt (hunk 1 above)
Comment (aliernfrog) [reply src/main/kotlin/app/revanced/patches/youtube/misc/zoomhaptics/annotations/ZoomHapticsCompatibility.kt]: Done, removed versions array

**Review by oSumAtrIX — APPROVED** [review]

**Comment by oSumAtrIX** [comment]
Thanks.

**Comment by aliernfrog** [comment]
https://github.com/revanced/revanced-integrations/pull/217 needs to be merged as well

**Pull request merged**